src/Network/Network/help/New-AzApplicationGatewayRewriteRuleCondition.md

```diff
@@ -0,0 +1,135 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azapplicationgatewayrewriterulecondition
+schema: 2.0.0
+---
+
+# New-AzApplicationGatewayRewriteRuleCondition
+
+## SYNOPSIS
+Adds a condition to the RewriteRule for an application gateway.
+
+## SYNTAX
+
+```
+New-AzApplicationGatewayRewriteRuleCondition -Variable <String> [-Pattern <String>] [-IgnoreCase] [-Negate]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+**The AzApplicationGatewayRewriteRuleCondition** cmdlet creates a rewrite rule condition for an Azure application gateway.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $condition = New-AzApplicationGatewayRewriteRuleCondition -Variable "var_request_uri" -Pattern "http" -IgnoreCase
```

Comment:
You should show the output when $condition is written to the screen as well.

Reply:
Done